src/components/InfoPanel/InfoPanel.js

```diff
@@ -53,7 +54,15 @@ const InfoPanel = () => {
           <Topology width={300} height={300} modelData={modelStatusData} />
           <i
             className="p-icon--expand"
-            onClick={() => setShowExpandedTopology(!showExpandedTopology)}
+            onClick={() => {
+              setShowExpandedTopology(!showExpandedTopology);
+              if (showExpandedTopology) {
```

Comment:
This should be the same value as being set on 58, yes? As at this point in the code the `showExpandedTopology` will be the old value.

Reply:
Because it's not in the callback, good catch.